db.json

```diff
@@ -1,15 +0,0 @@
-{
```

Comment:
You put db.json on the .gitignore
Nice :)

Reply:
Thanks ;) ...